tests/code/test_course_management.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from app.models import mainDB
+from app.logic.courseManagement import *
+from app.models.course import Course
+from app.models.courseInstructor import CourseInstructor
+
+@pytest.mark.integration
+def test_course_management():
+    with mainDB.atomic() as transaction:
+        testingCourse = Course.create(courseName = "Testing Approved",
+                                        term = 3,
+                                        status = 1,
+                                        courseCredit = "7",
+                                        createdBy = "Mayjue",
+                                        isAllSectionsServiceLearning = 0,
+                                        isPermanentlyDesignated = 0)
+
+        testingCourse = Course.create(courseName = "Testing Pending",
+                                        term = 3,
+                                        status = 2,
+                                        courseCredit = "12",
+                                        createdBy = "Tyler Parton",
+                                        isAllSectionsServiceLearning = 0,
+                                        isPermanentlyDesignated = 0)

```

Comment:
Why not add these to the test data, instead of specifying it all here?

Reply:
Also, the test fails for me. `/Users/ramsayb2/dev/celts/tests/code/test_course_management.py:46: AssertionError: assert 'Testing Approved' in []`

Reply:
I ran the test and It passes for me, not sure if there is a difference in test data causing it, if so I don't know what that would be. Regarding the reason we added the creations there, the idea was regardless of how someone manipulated the test data out test would pass. If we added it to test data the issue is someone can go in and change it which would fail our test. We figured the best way to rectify this was to have it create and delete the event within the test and regardless of what someone had in their own branch it would pass. @BrianRamsay